### PR TITLE
Move skip link top so it fits within brand banner

### LINKED ISSUE #2944: Skip to content link overflows brand container < 400px
**Describe the bug**
We reduced the height of the brand container on screen widths below 400px, and now the skip to content link doesn't fit inside:

Screenshot 2020-01-16 at 11 49 02

This could be fixed without reducing the tappable area by changing the 'top' value from 0.5rem to 0 at this breakpoint (check with UX if this is an acceptable solution):

Screenshot 2020-01-16 at 11 49 23

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 'http://localhost:7080/afrique'
2. Tab through to skip to content link

**Expected behaviour**
The skip to content link should not overflow its container.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by tochwill**
Resolves #2944

**Overall change:** Move skip to content link top value to stay within brand banner with height <400px.

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-brand/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 5.1.4 | [PR#XXXX](https://github.com/bbc/psammead/pull/XXXX) Move skip to content link top to stay within brand banner height |
```
Comment (tochwill) [inline packages/components/psammead-brand/CHANGELOG.md]:
```suggestion
| 5.1.4 | [PR#2946](https://github.com/bbc/psammead/pull/2946) Move skip to content link top to stay within brand banner height |
```

**Review by DenisHdz — APPROVED** [review]
Thanks for the fix 👍

**Review by DenisHdz — CHANGES_REQUESTED** [review]
You have to bump the `package-lock.json` too.

**Review by AlistairGempf — APPROVED** [review]
Looks good

**Review by DenisHdz — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Looks good to me..

**Comment by tochwill** [comment]
Just need to update package.json etc after resolving conflicts in the changelog

**Pull request merged**